inc/saiacl.h

```diff
@@ -227,21 +254,149 @@ typedef enum _sai_acl_table_group_attr_t
     SAI_ACL_TABLE_GROUP_ATTR_START,
 
     /**
-     * @brief Dummy attribute
+     * @brief ACL stage
      *
-     * @type sai_int32_t
-     * @flags READ_ONLY
+     * @type sai_acl_stage_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_ACL_TABLE_GROUP_ATTR_ACL_STAGE = SAI_ACL_TABLE_GROUP_ATTR_START,
+
+    /**
+     * @brief List of ACL bind points where this group will be applied
+     *
+     * @type sai_s32_list_t sai_acl_bind_point_t
+     * @flags CREATE_ONLY
+     * @default empty
      */
-    SAI_ACL_TABLE_GROUP_ATTR_DUMMY = SAI_ACL_TABLE_GROUP_ATTR_START,
+    SAI_ACL_TABLE_GROUP_ATTR_ACL_BIND_POINT_LIST,
```

Comment:
@lguohan as we discussed - this will simplify the implementation. ACL groups act as a container where we can hold various acl tables. The user's intention to simply specifiy what bind points any specific acl group is going to be associated -- at creation time gives a better error checking/handling when new acl tables (via acl group members) are added to the groups. It makes the sai implementation management simple.